### PR TITLE
login & signup page deisgn according to new theme

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/login/login.component.html [hunk 1]
```diff
@@ -1,276 +1,188 @@
...
+        <div class="row mx-0">
+            <div class="col-md-6 col-lg-6 col-sm-6 col-12 px-0">
+                <div class="login-left">
+                </div>
+            </div>
+            <div class="col-md-6 col-lg-6 col-sm-6 col-12 px-0">
+                <div class="login-right d-flex align-items-center ">
+                    <div class="giddh-login">
+                        <div class="giddh-login-inner">
+                            <div id="infoSpan"></div>
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.html]: remove blank div: <div id="infoSpan"></div>

File: apps/web-giddh/src/app/login/login.component.html [hunk 2]
```diff
@@ -1,276 +1,188 @@
...
-                                <div class="col-xs-12">
-                                </div>
+    <div class="login-box">
+        <div class="row mx-0">
+            <div class="col-md-6 col-lg-6 col-sm-6 col-12 px-0">
+                <div class="login-left">
+                </div>
+            </div>
+            <div class="col-md-6 col-lg-6 col-sm-6 col-12 px-0">
+                <div class="login-right d-flex align-items-center ">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.html]: Remove space

File: apps/web-giddh/src/app/login/login.component.html [hunk 3]
```diff
@@ -1,276 +1,188 @@
...
+                            <div id="infoSpan"></div>
+                            <!-- dynamic content inserted -->
+                            <div class="logo-login">
+                                <a href="https://giddh.com">
+                                    <img src="./assets/images/giddh-white-logo.svg" alt="" width="141px">
+                                </a>
+                            </div>
+                            <div class="accounting-space bg-white">
+                                <section id="fullView">
+                                    <h4 class="mrB3">Login to your secure accounting space</h4>
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.html]: mrB3 to mb-3

File: apps/web-giddh/src/app/login/login.component.html [hunk 4]
```diff
@@ -1,276 +1,188 @@
...
+                        <div class="giddh-login-inner">
+                            <div id="infoSpan"></div>
+                            <!-- dynamic content inserted -->
+                            <div class="logo-login">
+                                <a href="https://giddh.com">
+                                    <img src="./assets/images/giddh-white-logo.svg" alt="" width="141px">
+                                </a>
+                            </div>
+                            <div class="accounting-space bg-white">
+                                <section id="fullView">
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.html]: id name should be full-view

File: apps/web-giddh/src/app/login/login.component.html [hunk 5]
```diff
@@ -1,276 +1,188 @@
...
+                                    <h4 class="mrB3">Login to your secure accounting space</h4>
+                                    <form class="pos-rel mt-2 mb-2" id="loginWithPwd" [formGroup]="loginWithPasswdForm"
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.html]: id name should be login-with-pwd

File: apps/web-giddh/src/app/login/login.component.html [hunk 6]
```diff
@@ -1,276 +1,188 @@
...
+                                        </div>
+                                        <a href="javascript:void(0)" class="pull-right btn-link fs14 forget-pw"
+                                            (click)="showForgotPasswordModal()">Forgot
+                                            password?</a>
+                                    </form>
+                                    <form class="mt-2 mb-2" *ngIf="isLoginWithPasswordSuccessNotVerified$ | async"
+                                        id="signupVerifyForm" [formGroup]="signupVerifyForm" autocomplete="off">
+                                        <div class="clearfix">
+
+                                            <input class="form-control mrB1" formControlName="verificationCode"
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.html]: in the all classes name should be mb-1

File: apps/web-giddh/src/app/login/login.component.scss [hunk 7]
```diff
@@ -214,35 +213,22 @@ button.close {
...
+    border-color: $midnightBlue;
 }
 
-:host ::ng-deep .btn-success:hover {
-    background: #2c2c43;
-    /*  */
+.btn-success:hover {
+    background: $midnightBlueHover;
+    border-color: $midnightBlueHover;
+    color: #fff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.scss]: use color var

File: apps/web-giddh/src/app/login/login.component.scss [hunk 8]
```diff
@@ -255,7 +241,7 @@ h1 {
     font-size: 24px;
 }
 
-.slide_content {
+.slide-content {
     padding: 80px 100px;
     color: #fff;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.scss]: use color var

File: apps/web-giddh/src/app/login/login.component.scss [hunk 9]
```diff
@@ -264,17 +250,17 @@ h1 {
     justify-content: center;
 }
 
-.login_box {
+/* .login-box {
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.scss]: why comment on this code?

File: apps/web-giddh/src/app/login/login.component.scss [hunk 10]
```diff
@@ -264,17 +250,17 @@ h1 {
...
     background: #fff8ea url("/assets/images/new/banner-grid-design.png") repeat;
     padding: 20px 70px;
-}
+} */
 
-.login_box h1 {
+.login-box h1 {
     color: #0f0f27;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.scss]: use color var

File: apps/web-giddh/src/app/login/login.component.scss [hunk 11]
```diff
@@ -284,52 +270,54 @@ small {
     color: #333;
```
Comment (sadik-walkover) [inline apps/web-giddh/src/app/login/login.component.scss]: use color var

File: apps/web-giddh/src/app/login/login.component.scss [hunk 12]
```diff
@@ -348,167 +331,227 @@ small {
...
+    color: $pageTitleColor;
+    font-size: 0.9em;
+}
+.login-with a {
+    color: $pageTitleColor;
+    text-decoration: none;
+}
+.btn-secondary {
+    background-color: $White;
+    color: #000962;
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/login/login.component.scss]: Define var for this color

File: apps/web-giddh/src/app/login/login.component.scss [hunk 13]
```diff
@@ -348,167 +331,227 @@ small {
...
+    border: 1px solid $lightTextBlue;
+    border-radius: 5px;
+}
+.btn-secondary:hover {
+    color: $White !important;
+    background-color: $lightTextBlue;
+    border-color: $lightTextBlue;
+}
+.forgotStep {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/login/login.component.scss]: Should be dash-case

**Comment by sadik-walkover** [comment]
Some conflict in this file, pls check

**Pull request merged**